rpcs3/Emu/RSX/VK/VKProgramBuffer.h

```diff
@@ -206,6 +206,11 @@ class VKProgramBuffer : public program_state_cache<VKTraits>
 	void add_pipeline_entry(RSXVertexProgram &vp, RSXFragmentProgram &fp, vk::pipeline_props &props, Args&& ...args)
 	{
 		props.render_pass = m_render_pass_data[props.render_pass_location];
+		if (props.render_pass == 0)
```

Comment:
given how pointless the render pass id is, just replace these with a `verify("Usupported renderpass configuration" HERE), render_pass_id != VK_NULL_HANDLE;`